lib/api/bucketGetVersioning.js

```diff
@@ -15,23 +14,25 @@ import services from '../services';
  * @return {string} - the converted xml string of the versioning configuration
  */
 function convertToXml(versioningConfiguration) {
-    const options = { declaration: { standalone: 'yes', encoding: 'UTF-8' } };
-    const constructedJSON = {
-        VersioningConfiguration: [
-            { _attr: { xmlns: 'http://s3.amazonaws.com/doc/2006-03-01/' } },
-        ],
-    };
+    const xml = [];
+
+    xml.push('<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
+             '<VersioningConfiguration ' +
+                'xmlns="http://s3.amazonaws.com/doc/2006-03-01/">'
+    );
+
     if (versioningConfiguration && versioningConfiguration.Status) {
-        constructedJSON.VersioningConfiguration.push(
-            { Status: versioningConfiguration.Status });
+        xml.push(`<Status>${versioningConfiguration.Status}</Status>`);
```

Comment:
Yes, it appears that the only possible values of `versioningConfiguration.Status` and  `versioningConfiguration.MfaDelete` are `undefined`, `'Enabled'`, and `'Suspended'`.
See: https://github.com/scality/S3/blob/master/lib/metadata/BucketInfo.js#L63

Thus, no need for escaping.